### ISSUE TITLE
Use Widoco for documentation

### ISSUE THREAD

**Issue by pwin** [semanticarts/gist#170]
Could be included in the build process

Would give scope for including chunks of HTML for abstract, introduction, etc

Gives scope for URI for each class/property

See https://github.com/dgarijo/Widoco

**Comment by uscholdm** [comment]
Good idea. Morgan Stanley has been using this for a while.  Could be added to our publish process.

**Comment by pwin** [comment]
Yes, I've used it before and have started my own gist documentation using
Widoco too.  I think that the one area of challenge will be documenting
which module the classes/props are described in, but I'm sure that is
something we can discuss with Daniel (Widoco's author) if it turns out to
be non-trivial
